web/Dockerfile

```
@@ -1,50 +1,64 @@
-# Args and Environment Variables
 ARG GECKOVERSION=0.33.0
 ARG GOVERSION=1.21.5
-ENV DEBIAN_FRONTEND="noninteractive"
-ENV DATABASE="postgres"
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
-ENV GOROOT="/usr/local/go"
-ENV GOPATH=$HOME/go
-ENV PATH="${PATH}:${GOROOT}/bin:${GOPATH}/bin"
+
+RUN ARCH=$(dpkg --print-architecture) \ 
```

Hadolint

[error] Expected test to end here (don't wrap commands in []/[[]]). Fix any mentioned problems and try again.

(SC1072)